content/sensu-core/2.0/reference/rbac.md

```diff
@@ -10,117 +10,200 @@ menu:
-
-To remove an organization from the system, run the `delete` subcommand.
-
-{{< highlight shell >}}
-sensuctl organization delete acme
-{{< /highlight >}}
-
-## Environments
-
-The initial installation of Sensu includes a `default` environment,
-included in the `default` organization.
-
-You can create additional environments to map your development workflow.
-
-### Viewing Environments
-To view all the environments currently configured in your Sensu installation,
-run the following:
+A user represents a person or an agent which interacts with Sensu.
 
-{{< highlight shell >}}
-sensuctl environment list
-{{< /highlight >}}
+#### Attributes
+
+username     | 
+-------------|------ 
+description  | The name of the user. Cannot contain special characters.
+required     | true 
+type         | String
+example      | {{< highlight shell >}}"name": "admin"{{</highlight>}}
+
+password     | 
+-------------|------ 
+description  | The user's password. Cannot be empty. 
+required     | true 
+type         | String
+example      | {{< highlight shell >}}"password": "P@ssw0rd!"{{</highlight>}}
+
+roles        | 
+-------------|------ 
+description  | A list of roles the user belongs to.
+required     | false 
+type         | Array
+example      | {{< highlight shell >}}"roles": ["admin"]{{</highlight>}}
+
+disabled     | 
+-------------|------ 
+description  | The state of the user's account.
+required     | false 
+type         | Boolean 
+default      | false
+example      | {{< highlight shell >}}"disabled": false{{</highlight>}}
+
+### Group
 
-### Managing Environments
+Not available yet!
 
-To add a new environment, simply call the `sensuctl` environment create
-command with it's intended name.
+### Role
 
-{{< highlight shell >}}
-sensuctl environment create dev
-{{< /highlight >}}
+A role contains a set of rules, which represent permissions to Sensu resources.
 
-To remove an environment from the system, run the `delete` subcommand.
+Roles can be assigned to one or multiple users. Each user can be a member of one
+or multiple roles. Users inherit all of the permissions from each role they are
+in.
 
-{{< highlight shell >}}
-sensuctl environment delete dev
-{{< /highlight >}}
+The initial installation of Sensu includes an initial `admin` role that gives a
+user associated with it full access to the system.
+
+#### Attributes
+
+name         | 
+-------------|------ 
+description  | Name of the role 
+required     | true 
+type         | String
+example      | {{< highlight shell >}}"name": "admin"{{</highlight>}}
+
+rules        | 
+-------------|------ 
+description  | The rulesets that a role applies.
+required     | true 
+type         | array 
+example      | {{< highlight shell >}}"rules": [
+    {
+        "type": "*",
+        "environment": "*",
+        "organization": "*",
+        "permissions": [
+            "create",
+            "read",
+            "update",
+            "delete"        
+        ]
+    } 
+]{{</highlight>}}
+
```

Comment:
Could you add a full example here? Nothing complex, the `read-only` role could be perfect!

Reply:
Do you mean for rule or role?

Reply:
Oh nvm, couldn't see which line the comment was on until I switched to the inline view *shakes fist at github*